tests/sentry/sentry_metrics/querying/test_api.py

```diff
@@ -28,11 +28,11 @@ def setUp(self):
 
         for value, transaction, platform, env, time in (
             (1, "/hello", "android", "prod", self.now()),
-            (3, "/hello", "ios", "dev", self.now()),
+            (6, "/hello", "ios", "dev", self.now()),
```

Comment:
Changed the values to make the order by always with different values (otherwise the test execution time could lead to a non-deterministic ordering of the same valued rows).